scripts/zones/Rolanberry_Fields_[S]/mobs/Delicieuse_Delphine.lua

```diff
@@ -1,9 +1,36 @@
 -----------------------------------
 -- Area: Rolanberry Fields [S]
 --   NM: Delicieuse Delphine
+-- https://www.bg-wiki.com/ffxi/Delicieuse_Delphine
+-- Only uses Impale, and does it in threes
+-- TODO allow deaggro based on distance (core CMobEntity::CanDeaggro() forces NM and Battlefield mobs to never stop chasing)
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMod(xi.mod.DOUBLE_ATTACK, 5)
+    mob:setMod(xi.mod.TRIPLE_ATTACK, 50)
+    mob:addMod(xi.mod.MOVE_SPEED_STACKABLE, 18)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+end
+
+entity.onMobWeaponSkillPrepare = function(mob, target)
```

Comment:
prepare is only called when it readies a TP move on TP, and isn't queued up from useMobAbility?

Reply:
Correct, it's only called when the controller decides which skill to use

So `mob:useMobAbility()` would call it, but passing a parameter would not